src/app/components/home/home.component.html

```diff
@@ -0,0 +1,34 @@
+<div class="home-container">
+    <div class="home-top">
+        <h1 class="add-task-label">Add Task</h1>
+        <div class="home-top-buttons">
+            <div class="create-button">
+                <button (click)="showAddTaskCard()">＋ Create</button>
+            </div>
+            <div class="filter-buttons">
+
+            </div>
+        </div>
+    </div>
+    <div class="task-cards-container" >
```

Comment:
let's move these to separate components.

Reply:
I have created a 'home-top' component and separated this portions there.